src/main/java/com/minecolonies/coremod/blocks/BlockHutDeliveryman.java

```diff
@@ -1,5 +1,8 @@
 package com.minecolonies.coremod.blocks;
 
+import com.minecolonies.coremod.tileentities.TileEntityColonyBuilding;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.world.World;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.world.World'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)